src/main/java/org/ow2/proactive/connector/iaas/cloud/provider/azure/AzureProviderNetworkingUtils.java

```diff
@@ -243,8 +236,8 @@
 
         // Configure public IP address
         NetworkInterface.DefinitionStages.WithCreate networkInterfaceCreationStage4 = Optional.ofNullable(optionalPublicIpAddress)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "networkInterfaceCreationStage4". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1488)